book/notes.qmd

```diff
@@ -1,4 +1,5 @@
 # Notes
+
 <!-- Note that for images the relative path doesn't work since moved the book to separate repo and automated quarto publishing -->
 <!-- links like "![](./../images/tynewear_lads_msoas.png)" nor "![](images/tynewear_lads_msoas.png)"" (as done before) wouldn't work-->
 <!-- followed the suggestions from here  https://stackoverflow.com/questions/51002481/images-not-loading-when-deploying-to-github-pages-->
```

Comment:
Many of the images inside `notes.qmd` return a 404 error. I think that if you want to include them, then you have to shift the `images` directory inside the `book` directory. Then, you can use a relative path.

However, more broadly speaking, I question whether this page is actually useful for a general audience? Firstly, it's not very up to date. Secondly, it's not very understandable for someone who wasn't in the relevant meetings/discussions. (I understand that that was probably not the intention when writing these!)

If you don't think that people need to be reading this, I would propose to remove it entirely. If you want to preserve the content, I would move it to a separate directory of the repository which is not published with the Quarto book.

Reply:
It would feel in general more correct to move the whole project in a Turing or UA repo, as it is now in a more mature phase and, I think, also ready for the big jump!

Reply:
I would simply keep the markdown file around for a reference but remove it from the book TOC.

Reply:
Ok, I've removed it from the ToC and added a note in the README saying that it exists (so that it doesn't get forgotten about).

Reply:
More generally, regarding where to put the repos ... we still haven't quite decided on that, have we. I'll bring it up at tomorrow's meeting again.